resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -4,112 +4,346 @@
 ---
 
 - slug: feature_counts
-  name: FeatureCounts
+  name: featureCounts
   requirements:
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/legacy:latest
-  data_name: "{{ alignments.bam.file|basename|default('?') }} ({{ (alignments|sample_name) }})"
-  version: 0.3.2
+        image: resolwebio/rnaseq:2.1.1
+    resources:
+      cores: 1
+  data_name: "{{ alignment.reads|sample_name|default('?') }}"
+  version: 0.3.3
   type: data:expression:featurecounts
   category: analyses
   flow_collection: sample
   persistence: CACHED
   description: >
-    A highly efficient and accurate read summarization program.
+    An ultrafast and accurate read summarization program. First samtools are
+    used to determine, if aligned reads are single or paired-end. After that
+    the featureCounts tool produces expressions according to the parameters
+    set in the process inputs. Finally, raw expression counts are normalized
+    by computing TPM and FPKM values. See
+    [the official website](http://bioinf.wehi.edu.au/featureCounts/) and
+    [the introductory paper](https://academic.oup.com/bioinformatics/article/30/7/923/232889)
+    for more information.
   input:
-    - name: alignments
-      label: Aligned reads
-      type: data:alignment:bam
+    - name: alignment
+      label: Alignment
+      group:
+        - name: reads
```

Comment:
We usually use `reads` for raw .fastq input data. I think the original choice to have this as `Aligned reads` is better. Alternatively, think about using `Mapping (BAM)` or `Alignment` or something even better here.